### PR TITLE
🐛 #140 fix dateToFilterBy date

### LINKED ISSUE #140: Punishments occasionally not loading
Punishments do not always load when loading the page. Could be a bug in localStorage.

- [x] punishments should display correctly and work as expected.

196414672-e93c9c67-3cf0-4497-b6d9-594290f2afd7

### PR DESCRIPTION
**Pull request by VigdisIS**
Fixed that all punishments show. 
It still happens that one needs to refresh 1-2 times before the users load in the grid, due to localStorage etc., however this method might change once we generate users from OW group, so this could be fixed in a separate issue/pr.

Closes #140

### CODE REVIEW

File: frontend/src/components/punishments/PunishmentsListed.svelte [hunk 1]
```diff
@@ -14,22 +16,26 @@
...
+  {#await row.straffer}
+    <Circle size="60" color="#153E75" unit="px" duration="1s" />
+  {:then}
+    {#if row.straffer}
+      {#each row.straffer
+        .filter((pun) => $punishmentsToFilter
+            .map((pun) => pun.punishment_type_id)
+            .includes(pun.punishment_type))
+        .filter((pun) => ($showPaid ? pun : pun.verified_time === null))
+        .filter( (pun) => shouldDisplay(new Date(pun.created_time), $onlyShowAfterDate, $onlyShowBeforeDate) ) as punishment}
```
Comment (Terbau) [inline frontend/src/components/punishments/PunishmentsListed.svelte:28]: ocd go brrr

**Review by Terbau — CHANGES_REQUESTED** [review]
I hereby request extremely important changes!!

File: frontend/src/components/punishments/PunishmentsListed.svelte (hunk 1 above)
Comment (VigdisIS) [reply frontend/src/components/punishments/PunishmentsListed.svelte:28]: prettier autoformatter den til ( ( wth :<

**Review by annaiandresen — APPROVED** [review]
nicenice

**Pull request merged**